src/modules/main/app.controller.ts

```diff
@@ -35,4 +44,73 @@ export class AppController {
       identityPoolId: this.configService.get('IDENTITY_POOL_ID')
     };
   }
+
```

Comment:
Can we add a `GET` also for `ssm-params`,  to list all ssm params  ?

Reply:
Yes, we can and I will do that ASAP.